config/locales/en.yml

```diff
@@ -0,0 +1,33 @@
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
+#
+# To use the locales, use `I18n.t`:
+#
+#     I18n.t 'hello'
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
+# To learn more, please read the Rails Internationalization guide
+# available at https://guides.rubyonrails.org/i18n.html.
+
+en:
+  hello: "Hello world"
```

Comment:
I guess this won’t conflict with our existing translations?

Reply:
It seems that it doesn't.

```
[1] pry(main)> I18n.default_locale
=> :"en-US"
[2] pry(main)> I18n.t('bgeigie_imports.delete.delete_this_import')
=> "Delete this Import"
[3] pry(main)> I18n.locale = :ja
=> :ja
[4] pry(main)> I18n.t('bgeigie_imports.delete.delete_this_import')
=> "このインポートを削除する"
[5] pry(main)> I18n.t('hello')
=> "こんにちは"
[6] pry(main)> I18n.locale = :'en-US'
=> :"en-US"
[7] pry(main)> I18n.t('hello')
=> "Hello"
[8] pry(main)> I18n.locale = :en
=> :en
[9] pry(main)> I18n.t('hello')
=> "Hello world"
[10] pry(main)> I18n.t('bgeigie_imports.delete.delete_this_import')
=> "Delete this Import"
```


[2.1 Configure the I18n Module
](https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module) says in the following, but I don't understand what it means..

> The i18n library takes a pragmatic approach to locale keys...

Reply:
Yeah I realized also our acceptance specs would fail if it was a problem. Looks good!